ui/apps/platform/src/Containers/MainPage/AnnouncementBanner.tsx

```diff
@@ -0,0 +1,42 @@
+import React, { ReactElement, useEffect, useState } from 'react';
+import { AlertVariant, Banner, Button } from '@patternfly/react-core';
+
+const ANNOUNCEMENT_BANNER_KEY = 'postgresAnnouncementBannerDismissed';
+
+function AnnouncementBanner(): ReactElement | null {
+    const [isDisplayed, setIsDisplayed] = useState(false);
+
+    useEffect(() => {
+        const localStorageValue = localStorage.getItem(ANNOUNCEMENT_BANNER_KEY);
+        const isBannerDismissed = localStorageValue
+            ? Boolean(JSON.parse(localStorageValue))
+            : false;
```

Comment:
nit: In future, wrap `JSON.parse` calls in `try... catch...`

Trust no one.

Reply:
Good point. After all was said and done, line 12 could have been `localStorageValue === 'true'` correct?